core/storage/suggestion/gae_models.py

```diff
@@ -917,6 +917,21 @@ def get(cls, language_code, contributor_user_id, topic_id):
             language_code, contributor_user_id, topic_id)
         return cls.get_by_id(entity_id)
 
+    @classmethod
+    def get_all_by_user_id(cls, user_id):
+        """Gets all TranslationContributionStatsModels matching the supplied
+        user_id.
+
+        Returns:
+            list(TranslationContributionStatsModel). The matching
+            TranslationContributionStatsModels.
+        """
+        return (
+            cls.get_all()
+            .filter(cls.contributor_user_id == user_id)
+            .fetch(feconf.DEFAULT_QUERY_LIMIT)
```

Comment:
Does this need to be paged?

Reply:
I figured the number of distinct (language_code, topic_id) model instances per user would be very unlikely to exceed 1000. Alternatively, we could perhaps order by date so that we retrieve only the latest contributions.

Reply:
Ah ok. No strong objections -- happy to defer to you and Ryan on this, just wanted to make sure it's been thought about.